schemas/json/font-collection.json

```diff
@@ -138,6 +138,12 @@
 								}
 							}
 						},
+						"required": [
+							"name",
+							"fontFamily",
+							"slug",
+							"fontFace"
```

Comment:
I think these four are required in the `font_family_settings` field, but are they correct?

Reply:
nope, fontFace is not required

Reply:
Thanks for the review! Updated by 9e82875.